src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Layers/FeatureLayerDefinitionExpression/FeatureLayerDefinitionExpression.xaml.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright 2016 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an 
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific 
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Data;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using System;
+using System.Windows;
+using System.Windows.Controls;
+
+namespace ArcGISRuntime.Desktop.Samples.FeatureLayerDefinitionExpression
+{
+    public partial class FeatureLayerDefinitionExpression : UserControl
```

Comment:
Please remove : UserControl and needed using if that is not anymore used. This is not needed in the code since the base type is already defined in xaml. Please make sure that this is done for all samples. This is related on generating individual samples for the API reference.